npc/re/quests/illusion_investigation.txt

```diff
@@ -0,0 +1,1559 @@
+		next;
+		mes "[Jiroker]";
+		mes "Actually, I'm the one who's more timid than anyone else... heck";
+		next;
+		mes "[Jiroker]";
+		mes "Anyway... I'm in charge of investigating this place.";
+		next;
+		mes "[Jiroker]";
+		mes "Have you heard about the investigation from Captain Joel? Adventurer, you have to completely hunt 100 monsters by yourself.";
+		next;
+		mes "[Jiroker]";
+		mes "Would you like to participate in the investigation of this place?";
+		next;
+		if (select( "Participate", "Do not participate" ) == 2) {
+			mes "[Jiroker]";
+			mes "Then should I ask another adventurer...";
+			close;
+		}
+		mes "[Jiroker]";
+		mes "I like it!!";
+		next;
+		mes "[Jiroker]";
+		mes "Just kill 100 monsters of any type here.";
+		next;
+		mes "[Jiroker]";
+		mes "Then be careful!";
+		close2;
+		setquest 3446;
+		completequest 3446;
+		setquest 3447;
+		end;
+	}
+	switch( checkquest(3448,PLAYTIME) ) {
+	case -1:
+		break;
+	case 0:
+	case 1:
+		mes "[Jiroker]";
+		mes "Ah, that's enough for today's investigation.";
+		next;
+		mes "[Jiroker]";
+		mes "It will be restored tomorrow, so if you can afford it, I would appreciate it if you could come and help me with the investigation!!";
+		close;
+	case 2:
+		erasequest 3448;
+		break;
+	}
+	switch( checkquest(3447,HUNTING) ) {
+	case -1:
+		mes "[Jiroker]";
+		mes "Hello, are you here today to help with the investigation?";
+		next;
+		mes "[Jiroker]";
+		mes "The content is the same as last time.";
+		next;
+		mes "[Jiroker]";
+		mes "If you kill 100 monsters of any type, would you like to participate?";
+		next;
+		if (select( "Participate", "Do not participate" ) == 2) {
+			mes "[Jiroker]";
+			mes "It would be nice if you could help me today...";
+			close;
+		}
+		mes "[Jiroker]";
+		mes "I like it!!";
+		next;
+		mes "[Jiroker]";
+		mes "Just kill 100 monsters of any type here.";
+		next;
+		mes "[Jiroker]";
+		mes "Then be careful!";
+		close2;
+		setquest 3447;
+		end;
+	case 0:
+	case 1:
+		mes "[Jiroker]";
+		mes "Kill 100 monsters of any type.";
+		close;
+	case 2:
+		mes "[Jiroker]";
+		mes "You're back safely!";
+		next;
+		mes "[Jiroker]";
+		mes "Okay... Then hmm... I'll have to wait and see how much monsters spawn for a while.";
+		next;
+		mes "[Jiroker]";
+		mes "Leave the rest to me!! Thanks!!";
+		close2;
+		erasequest 3447;
+		setquest 3448;
+		getexp 2500000,2500000;
+		end;
+	}
+	end;
+
+OnInit:
+	questinfo( QTYPE_QUEST, QMARK_YELLOW, "BaseLevel >= 100 && isbegin_quest(3470) == 2 && isbegin_quest(3446) == 0" );
+
+	questinfo( QTYPE_EVENT2, QMARK_YELLOW, "checkquest(3447,HUNTING) == 2" );
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "isbegin_quest(3446) == 2 && checkquest(3448,PLAYTIME) == -1 && checkquest(3447,HUNTING) == -1" );
+	questinfo( QTYPE_DAILYQUEST, QMARK_YELLOW, "checkquest(3448,PLAYTIME) == 2" );
+	end;
+}
+
+ice_dun02,142,18,5	script	Elysia#elisia	4_F_OPERATION,{
```

Comment:
npc name is `엘리시아` sometimes it can be translated to Elysia but in RO there is a reference to this NPC Name with itemid: 7508, it's `엘리시아의 반지` it's translated to `Allysia's Ring` so i guess can use Allysia instead of Elysia?